src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -343,9 +347,9 @@ def _iterate_hit_tsv(self, es_search_hit, writer):
 
     def _iterate_hit_bdbag(self, es_search_hit, participants: MutableSet[str], sample_writer):
         hit_dict = es_search_hit.to_dict()
-
-        assert len(hit_dict['contents']['specimens']) == 1
-        assert len(hit_dict['contents']['cell_suspensions']) == 1
+        # Some files are not associated with specimens (e.g., PDFs) - skip them.
+        if 'specimens' not in hit_dict['contents'].keys():
+            return
         assert len(hit_dict['contents']['files']) == 1
```

Comment:
Remove. The one() call takes care of this.

Reply:
Removed.